install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml

```diff
@@ -98,7 +98,7 @@ data:
         - {{ "[[ .ProxyConfig.BinaryPath ]]" }}
         - --serviceCluster
         {{ "[[ if ne \"\" (index .ObjectMeta.Labels \"app\") -]]" }}
-        - {{ "[[ index .ObjectMeta.Labels \"app\" ]].[[ valueOrDefault .DeploymentMeta.Namespace \"default\" ]]" }}
+        - {{ "[[ index .ObjectMeta.Labels \"app\" ]]." }}$(POD_NAMESPACE)
```

Comment:
shouldn't we have this within the `""`?
Ie. {{ "[[ index .ObjectMeta.Labels \"app\" ]].$(POD_NAMESPACE)" }}

Reply:
Or maybe keep the `valueOrDefault` to be on the safe side:
{{ "[[ index .ObjectMeta.Labels \"app\" ]].[[ valueOrDefault $(POD_NAMESPACE) \"default\" ]]" }}

Reply:
When deployed to the 'default' namespace, POD_NAMESPACE is default - not sure it can be a blank value. I will do a quick test to confirm - but I believe I tried this before.

Reply:
Confirmed, POD_NAMESPACE is default when deployed to default namespace.